PhysicsTools/NanoAOD/python/l1trig_cff.py

```diff
@@ -0,0 +1,189 @@
+    cut = cms.string(""), 
+    name= cms.string("L1Tau"),
+    doc = cms.string(""),
+    extension = cms.bool(False), # this is the main table for L1 EGs
+    variables = cms.PSet(l1ObjVars,
+                         towerIEta = Var("towerIEta()",int,doc=""),
+                         towerIPhi = Var("towerIPhi()",int,doc=""),
+                         rawEt = Var("rawEt()",int,doc=""),
+                         isoEt = Var("isoEt()",int,doc=""),
+                         nTT = Var("nTT()",int,doc=""),                         
+                         hasEM = Var("hasEM()",int,doc=""),
+                         isMerged = Var("isMerged()",int,doc=""),
+
+                     )
+)
+
+l1EtSumTable = cms.EDProducer("SimpleTriggerL1EtSumFlatTableProducer",
+    src = cms.InputTag("caloStage2Digis","EtSum"),
+    minBX = cms.int32(-2),
+    maxBX = cms.int32(2),                           
+    cut = cms.string(""), 
+    name= cms.string("L1EtSum"),
+    doc = cms.string(""),
+    extension = cms.bool(False), 
+    variables = cms.PSet(l1PtVars,
+                         hwPt = Var("hwPt()",int,doc="hardware pt"),
+                         hwPhi = Var("hwPhi()",int,doc="hardware phi"),
+                         etSumType = Var("getType()",int,doc=""),
+                     )
+)
+
+l1EGTable = cms.EDProducer("SimpleTriggerL1EGFlatTableProducer",
+    src = cms.InputTag("caloStage2Digis","EGamma"),
+    minBX = cms.int32(-2),
+    maxBX = cms.int32(2),                           
+    cut = cms.string(""), 
+    name= cms.string("L1EG"),
+    doc = cms.string(""),
+    extension = cms.bool(False), 
+    variables = cms.PSet(l1ObjVars,
+                         towerIEta = Var("towerIEta()",int,doc="tower ieta"),
+                         towerIPhi = Var("towerIPhi()",int,doc="tower iphi"),
+                         rawEt = Var("rawEt()",int,doc="raw et"),
+                         isoEt = Var("isoEt()",int,doc="iso et"),
+                         footprintEt = Var("footprintEt()",int,doc=" footprint et"),
+                         nTT = Var("nTT()",int,doc="nr trig towers"),
+                         shape = Var("shape()",int,doc="shape"),
+                         towerHoE = Var("towerHoE()",int,doc="tower H/E"),
+                     )
+)
+
+l1TablesTask = cms.Task(l1EGTable,l1EtSumTable,l1TauTable,l1JetTable,l1MuTable)
+
+def setL1NanoToReduced(process):
+    """
+    sets the L1 objects only have reduced information which is necessary 
+    for central nano
+    """
+    #reduce the variables to the core variables
+    #note et sum variables are already reduced
+    process.l1EGTable.variables = cms.PSet(l1EGReducedVars)
+    process.l1MuTable.variables = cms.PSet(l1MuonReducedVars)
+    process.l1JetTable.variables = cms.PSet(l1JetReducedVars)
+    process.l1TauTable.variables = cms.PSet(l1TauReducedVars)
+    process.l1EtSumTable.variables = cms.PSet(l1EtSumReducedVars)
+   
+    #restrict bx
```

Comment:
Following the discussion, we have agreed to store all the BX's also in the reduced table for central prompt data Nano's

Reply:
thanks, will change